README.md

```diff
@@ -0,0 +1,51 @@
+Install postresql server
```

Comment:
The `README.md` file should contain other sections, as well. See e.g. https://gist.github.com/PurpleBooth/109311bb0361f32d87a2